spec.md

```diff
@@ -902,3 +903,102 @@ Platforms SHOULD initiate orphan mitigation in the following scenarios:
 | Timeout | Failure | Yes |
 
 If the platform marketplace encounters an internal error provisioning a service instance or binding (for example, saving to the database fails), then it MUST at least send a single delete or unbind request to the service broker to prevent creation of an orphan.
+
+## Service Metadata
```

Comment:
@jwforres I believe we have a couple we're using in OpenShift that should be suggested as possible conventional metadata content